### PR TITLE
Fix prompt unload (main)

### PR DESCRIPTION
**Pull request by joerunde**
See https://github.com/caikit/caikit-nlp/pull/282

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -16,6 +16,7 @@ classifiers=[
 dependencies = [
     "caikit[runtime-grpc,runtime-http]>=0.25.0,<0.26.0",
     "caikit-tgis-backend>=0.1.25,<0.2.0",
+    "caikit-tgis-backend>=0.1.27,<0.2.0",
```
Comment (evaline-ju) [inline pyproject.toml:19]: this seems to have become a duplicate line

Comment (markstur) [reply pyproject.toml:19]: for emphasis?

Comment (joerunde) [reply pyproject.toml:18]: (fixed)

**Review by evaline-ju — APPROVED** [review]
LGTM

**Pull request merged**